daemon/logger/splunk/splunk.go

```diff
@@ -163,6 +166,8 @@ func New(info logger.Info) (logger.Logger, error) {
 		return nil, err
 	}
 
+	logNonBlocking := info.Config["mode"] == "non-blocking"
```

Comment:
This should probably be using the `mode` `const` that was added.

Reply:
Fixed in a552c0b7abd2bbf2ba71d15fd8d72c98ce599a40